packages/localize/index.ios.ts

```diff
@@ -30,7 +30,12 @@ export function androidLaunchEventLocalizationHandler() {
 }
 
 export function overrideLocale(locale: string): boolean {
-	const path = NSBundle.mainBundle.pathForResourceOfType(locale.substring(0, 2), 'lproj');
+	let language_code = locale.substring(0, 2)
+	if (locale.indexOf('-') < 0 && locale.length == 3) {
+		language_code = locale
```

Comment:
This looks good. Please add semicolon on lines 33 and 35 for consistency sake as the rest of the file uses it.

Reply:
eslint precommit with husky should pikc this up automatically @shiv19 - will pull the pr to check